units/systemd-pcrlock-file-system.service.in

```diff
@@ -13,6 +13,7 @@ Documentation=man:systemd-pcrlock(8)
 DefaultDependencies=no
 Conflicts=shutdown.target
 Before=sysinit.target shutdown.target systemd-pcrlock-make-policy.service
+After=systemd-remount-fs.service
```

Comment:
hmm, indeed. should also have After=var.mount actually, because it locks the root fs UUID and /var fs UUID down in the PCR policy. But the latter is only going to work if /var/ is mounted, if it is split off.

Reply:
moreover if /var i ssplit off we need to wait for it anyway, because the file we want to write is also in /var.